ext4/internal/compactext4/compact.go

```diff
@@ -553,7 +567,7 @@ func (w *Writer) CreateWithParents(name string, f *File) error {
 		}
 		root = root.Children[dirname]
 	}
-	return w.Create(name, f)
+	return nil
```

Comment:
Why did you take this out?

Reply:
Earlier this function was used a wrapper for `Create`, this function used to first create all the parent directories and then call `Create`. Now, this function is always called on every path found in the tar and only then `Create` is called. So this function doesn't work like a wrapper on top of `Create` anymore. Does that make sense?

Reply:
Yep that makes sense!